openedx/core/djangoapps/celery_utils/management/commands/tests/test_cleanup_resolved_tasks.py

```diff
@@ -0,0 +1,73 @@
+"""
+Test management command to cleanup resolved tasks.
+"""
+
+from datetime import timedelta
+
+import ddt
+from django.test import TestCase
+from django.core.management import call_command
+from django.utils.timezone import now
+
+from openedx.core.djangolib.testing.utils import skip_unless_lms
+
+from .... import models
+
+DAY = timedelta(days=1)
+MONTH_AGO = now() - (30 * DAY)
+
+
+@ddt.ddt
+@skip_unless_lms
+class TestCleanupResolvedTasksCommand(TestCase):
+    """
+    Test cleanup_resolved_tasks management command.
+    """
+
+    def setUp(self):
+        self.task_ids = [
+            u'00000000-0000-0000-0000-000000000000',
+            u'11111111-1111-1111-1111-111111111111',
+            u'22222222-2222-2222-2222-222222222222',
+            u'33333333-3333-3333-3333-333333333333',
+        ]
```

Comment:
Sure.  But for unit testing for this management command, celery isn't called, I believe.